code/game/jobs/job/church.dm

```diff
@@ -81,7 +83,7 @@
 
 	stat_modifiers = list(
 	STAT_MEC = 25,
-	STAT_BIO = 10,
+	STAT_BIO = 25,
```

Comment:
If the chaplain's 35 gets brought down, the acolyte's 25 should also be brought down.